grails-app/services/org/pih/warehouse/data/LocationDataService.groovy

```diff
@@ -112,15 +124,23 @@ class LocationDataService {
             location = new Location()
         }
 
+        def locationType = params.locationType ? LocationType.findByNameLike(params.locationType + "%") : null
+        def currentLocationType = location.locationType
+
```

Comment:
Move this down to line 138 so the locationtype so we can keep it together with the actual processing.

Reply:
OK